assets/src/application/ui/display/TimezoneTimeInfo/index.tsx

```diff
@@ -33,16 +29,17 @@ const TimezoneTimeInfo: React.FC<OffsetInfoProps> = ({ className, date, inline =
 			<div className={'ee-focus-priority-6'}>{formatUtcDateForSite(date)}</div>
 		</div>
 	);
-	const htmlClassName = classNames(className, 'ee-timezone-info', { 'ee-timezone-info--inline': inline });
+	const label = __('click for timezone\ninformation');
+	const iconProps = {
+		icon: 'info-outline',
```

Comment:
have used `GlobalOutlined` from antd icons here, there was a type issue and ended up placing this temporary.
I have copied global outlined path, but all icons needs to be added in a separate PR as follows: https://chakra-ui.com/icon#adding-custom-icons